README.md

```diff
@@ -8,20 +8,22 @@ module "example-transfer" {
   source           = "github.com/schubergphilis/terraform-aws-mcaf-transfer"
   name             = "example"
   endpoint_type    = "VPC"
-  tags             = {}
-
   endpoint_details = {
     address_allocation_ids = ["eipalloc-12345", "eipalloc-67890"]
     subnet_ids             = ["subnet-12345", "subnet-67890"]
     vpc_id                 = "vpc-123456"
   }
+  restricted_mode = false
```

Comment:
Can you modify the Readme example according to the styleguide?:

    meta-argument: `count` or `for_each`
    meta-argument: `providers`
    <empty line>
    source
    version
    [alphabetically sorted other fields]
    <empty line>
    blocks (each must be separated by empty lines)
    <empty line>
    meta-argument blocks: (`depends_on`, `lifecycle`, etc)